srcpkgs/CalculiX-cgx/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'CalculiX-cgx'
+pkgname=CalculiX-cgx
+version=2.20
+revision=1
+wrksrc=CalculiX
+_cgx_dir=cgx_${version}
+build_style="gnu-makefile"
+make_build_args="-C ${_cgx_dir}/src"
+makedepends="glu-devel libglvnd-devel libXmu-devel libXi-devel"
+depends="tetgen"
+short_desc="3D Structural Finite Element Program - Graphical interface"
+maintainer="Phicem <phicem@gmx.com>"
+license="GPL-2.0-or-later, custom:GLUT"
+homepage="http://www.dhondt.de/"
+distfiles="http://www.dhondt.de/cgx_${version}.all.tar.bz2"
+checksum=f4c840c3633718547e33f86b2d7bc6b35608dd8d6cbf9fa192cecef34f2dc908
+if [ "$XBPS_MACHINE" == "x86_64" ] ; then
+	hostmakedepends+="glibc-devel-32bit"
+fi
+CFLAGS=" -O2 -Wall -Wno-narrowing  -I./  -I/usr/include/  -I/usr/include/GL  -I../../libSNL/src  -I../../glut-3.5/src  -I/usr/X11/include"
```

Comment:
```suggestion
CFLAGS="-DSEMINIT -I. -I${XBPS_CROSS_BASE}/usr/include/GL -I${XBPS_CROSS_BASE}/usr/X11/include -I../../libSNL/src -I../../glut-3.5/src"
```
only need to include the special cflags

Reply:
Hmm, I think I understand most of the suggested change here, but not all of it.

I understand that the `-02` flag is useless because it is already defined in `$XBPS_CFLAGS` and thus implicitely used.

I guess removing `-Wall` only suppresses warnings (which are not very useful in an automated build process), and can be safely removed, so I think I undestand this one as well. (I am correct?)

However I don't understand the removal of the `-Wno-narrowing` flag. I'm not very familiar with this flag but I thought it could remove errors (and not only warnings), and thus could change the build result. Why did you suggest removing this flag? Is it because it is considered that software should not be built when there are narrowing errors? Or is there another reason?

Reply:
I didn't notice any errors with it removed so I didn't think it was necessary to specify